internal/scaffold/cmd.go

```diff
@@ -44,10 +44,12 @@ import (
 	"fmt"
 	"os"
 	"runtime"
+	"strings"
 
 	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
```

Comment:
Move this import to the lower block.

Reply:
Why ? Should not be as:

- All static imports
- third-part ones ( k8s example )
- then, internal ones?

Reply:
Solved